mqboost/objective.py

```diff
@@ -105,6 +105,45 @@ def _hess_phuber(u: np.ndarray, alpha: float, delta: float) -> np.ndarray:
     return _hess
 
 
+def _grad_phuber2(u: np.ndarray, alpha: float, delta: float) -> np.ndarray:
+    """
+    Compute the gradient of the pseudo-Huber loss function.
+    Args:
+        u (np.ndarray): The error term.
+        alpha (float): The quantile level.
+        delta (float): The delta parameter.
+    Returns:
+        np.ndarray: The gradient of the pseudo-Huber loss function.
+    """
+
+    h = _grad_rho(u, alpha) ** 2 * u**2 / (delta**2)
+
+    _grad = -1 * (1 / np.sqrt(1 + h)) * abs(_grad_rho(u, alpha)) * u
+    return _grad
+
+
+# _grad_rho(u < 0).astype(int) - alpha
+
+
+def _hess_phuber2(u: np.ndarray, alpha: float, delta: float) -> np.ndarray:
+    """
+    Compute the Hessian of the pseudo-Huber loss function.
+    Args:
+        u (np.ndarray): The error term.
+        alpha (float): The quantile level.
+        delta (float): The delta parameter.
+    Returns:
+        np.ndarray: The Hessian of the pseudo-Huber loss function.
+    """
+    h = _grad_rho(u, alpha) ** 2 * u**2 / (delta**2)
+
+    _hess = (1 + h) ** (-3 / 2) / 2 * (abs(_grad_rho(u, alpha)) * u) ** 2 + (1 + h) ** (
+        -1 / 2
+    ) * (-1 * _grad_rho(u, alpha))
```

Comment:
Can you simplify this please? It's little bit confused.. 
```suggestion
    h = 1 + _grad_rho(u, alpha) ** 2 * u**2 / (delta**2)
    _hess = h ** (-3 / 2) / 2 * (abs(_grad_rho(u, alpha)) * u) ** 2 - h ** (-1 / 2) * _grad_rho(u, alpha)
```
BTW, can you explain `(-3/2)/2` this part? Is it just equls to `-3/4` or is there something I missed?